pkg/auth/cookie_manager.go

```diff
@@ -93,3 +94,28 @@ func (c CookieManager) SetTokenCookies(ctx context.Context, writer http.Response
 	}
 	return nil
 }
+
+func getLogoutAccessCookie() *http.Cookie {
+	return &http.Cookie{
+		Name:     accessTokenCookieName,
+		Value:    "",
```

Comment:
do you need to initialize these zero values?

Reply:
no, but may as well make this one explicit.  just because the value is set when you log in, I feel like I like explicitly unsetting it, but that's the irrational part of me talking.

Reply:
okie dokes